test/test-listing-alusd.spec.ts

```diff
@@ -0,0 +1,268 @@
+  });
+  
+  it('Should list correctly an asset: borrow on, collateral off, stable borrow off', async () => {
+
+    await (await gov.execute(proposal)).wait();
+    const proposalState = await gov.getProposalState(proposal);
+    expect(proposalState).to.be.equal(7);
+    const {
+      configuration: { data },
+      aTokenAddress,
+      stableDebtTokenAddress,
+      variableDebtTokenAddress,
+    } = await pool.getReserveData(TOKEN);
+    const poolData = parsePoolData(data);
+    expect(poolData).to.be.eql({
+      reserveFactor: RESERVE_FACTOR,
+      reserved: '0',
+      stableRateEnabled: '1',
+      borrowingEnabled: '1',
+      reserveFrozen: '0',
+      reserveActive: '1',
+      decimals: DECIMALS,
+      liquidityBonus: '0',
+      LiquidityThreshold: '0',
+      LTV: '0',
+    });
+
+    // preparing for tests.
+    aAlusd = (await ethers.getContractAt('IERC20', aTokenAddress, proposer)) as IERC20;
+    stableDebt = (await ethers.getContractAt('IERC20', stableDebtTokenAddress, proposer)) as IERC20;
+    variableDebt = (await ethers.getContractAt(
+      'IERC20',
+      variableDebtTokenAddress,
+      proposer
+    )) as IERC20;
+    await (await alusd.connect(alusdHolder).approve(pool.address, parseEther('200000'))).wait();
+    await (await aave.connect(proposer).approve(pool.address, parseEther('200000'))).wait();
+
+    // AAVE deposit by proposer
+    await (await pool.deposit(aave.address, parseEther('100'), proposer.address, 0)).wait();
+    // ALUSD deposit by ampl holder
+    const depositedAmount = parseEther('100').div(decimalMultiplier);
+    await (
+      await pool.connect(alusdHolder).deposit(alusd.address, depositedAmount, ALUSD_HOLDER, 0)
+    ).wait();
+    expect(await aAlusd.balanceOf(ALUSD_HOLDER)).to.gte(depositedAmount.sub(1));
+    expect(await aAlusd.balanceOf(ALUSD_HOLDER)).to.lte(depositedAmount.add(1));
+
+    // ALUSD holder not able to borrow DAI against ALUSD
+    await expect(
+      pool.connect(alusdHolder).borrow(dai.address, parseEther('1'), 2, 0, ALUSD_HOLDER)
+    ).to.be.revertedWith(ERRORS.NO_COLLATERAL_BALANCE);
+
+    // proposer able to borrow ALUSD variable against AAVE
+    const borrowAmount = parseEther('10').div(decimalMultiplier);
+    await (
+      await pool.connect(proposer).borrow(alusd.address, borrowAmount, 2, 0, proposer.address)
+    ).wait();
+    expect(await variableDebt.balanceOf(proposer.address)).to.be.equal(borrowAmount);
+
+    // proposer able to borrow ALUSD stable against AAVE
+    await (
+      await pool.borrow(alusd.address, borrowAmount, 1, 0, proposer.address)
+    ).wait();
+    increaseTime(40000);
+
+    // proposer able to repay ALUSD variable
+    await (await alusd.connect(proposer).approve(pool.address, parseEther('100000'))).wait();
+    await (await pool.repay(alusd.address, MAX_UINT_AMOUNT, 2, proposer.address)).wait();
+    expect(await variableDebt.balanceOf(proposer.address)).to.be.equal(parseEther('0'));
```

Comment:
Would be nice to add the repayment of ALUSD stable

Reply:
done